HealthcareSystem/Backend/Model/Users/Feedback.cs

```diff
@@ -18,6 +18,8 @@ public class Feedback
     {
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public DateTime SendingDate { get; set; }
```

Comment:
Why is SendingDate DatabaseGeneratedOption.Identity ?

Reply:
https://docs.microsoft.com/en-us/ef/core/modeling/generated-properties?tabs=data-annotations&fbclid=IwAR0__ecVDglOZEhuipolDDMPXQWNOLz-wK9RD5Grd8i0Au9x3CiqOnVAvXk